src/material/icon/icon-registry.ts

```diff
@@ -364,7 +366,7 @@ export class MatIconRegistry implements OnDestroy {
 
     // Not found in any cached icon sets. If there are icon sets with URLs that we haven't
     // fetched, fetch them now and look for iconName in the results.
-    const iconSetFetchRequests: Observable<SVGElement | null>[] = iconSetConfigs
+    const iconSetFetchRequests: Array<Observable<SVGElement | null>> = iconSetConfigs
```

Comment:
Use `(SVGElement | null)[]` instead of `Array`.

Reply:
I'm fine with either; Google's internal style guide says to use `Array<...>` when the generic is more than a single thing